apps/mobile/src/hooks/balances/use-stx-balances.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { AccountStore } from '@/store/accounts/utils';
+
+import { createMoney } from '@leather.io/utils';
+
+import { useStxBalance } from './use-stx-balances';
+
+// Mock the dependencies
+jest.mock('@leather.io/query');
```

Comment:
Why are we using jest? We're using vitest everywhere else

Reply:
I just generated these tests and hadn't seen. I will refactor to use `vitest` 👍